cypress/support/step_definitions/fireboltCalls.js

```diff
@@ -121,7 +121,7 @@ Given(/'(.+)' invokes the '(.+)' API to '(.+)'$/, async (appId, sdk, key) => {
         ? UTILS.getEnvVariable(CONSTANTS.THIRD_PARTY_APP_ID)
         : appId === CONSTANTS.THIRD_PARTY_APP
           ? UTILS.getEnvVariable(CONSTANTS.THIRD_PARTY_APP_ID)
-          : appId;
+          : UTILS.checkForSecondaryAppId(appId);
```

Comment:
add examples in glue description for secondary 3rd party app id too

Reply:
done